src/components/authorization/authorization.resolver.ts

```diff
@@ -0,0 +1,48 @@
+import { Args, Mutation, Query, Resolver } from '@nestjs/graphql';
+import { node } from 'cypher-query-builder';
+import { IdArg, ISession, Session } from '../../common';
+import { DatabaseService } from '../../core';
+import { Powers } from '../authorization/dto/powers';
+import { AuthorizationService } from './authorization.service';
+
+@Resolver()
+export class AuthorizationResolver {
+  constructor(
+    private readonly db: DatabaseService,
+    private readonly authorizationService: AuthorizationService
+  ) {}
+
+  @Query(() => [Powers])
+  async powers(@Session() session: ISession): Promise<Powers[]> {
```

Comment:
This really fulfills the TODO here
https://github.com/SeedCompany/cord-api-v3/blob/f3bd4b45e4b44975db656abede6ac1772857db50/src/components/authentication/authentication.dto.ts#L43

If we want to keep it here that's fine, but we should also add to LoginOutput and SessionOutput as it makes more sense there from a consumption perspective.